lib/lita-raindar.rb

```diff
@@ -7,6 +7,7 @@ class Raindar < Handler
       config :imgur_api_key
 
       route(/^weather ([a-zA-Z0-9\s]*)$/i, :radar, command: true, help: { "weather LOCATION" => "Returns GIF of recent weather radar" })
+      route(/^(radar locations)$/i, :radar_locations, command: true, help: { "radar locations" => "Returns list of available radar locations" })
```

Comment:
The `(` and `)` are only required if you need to extract the value in the handler, so I *think* it's safe to remove them from here